### DISCUSSION TITLE
[Feature] Shared album comments

### DISCUSSION THREAD

**Discussion by m-a-johnson** [damongolding/immich-kiosk#221] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Would love the ability to have comments on shared album photos be able to be displayed! Not sure what that would all involve, just think it would be neat to be able to see what was commented on a photo being displayed. 
Ideally, it would show up as like a bubble at the bottom or side of the screen with the users profile photo and their comment. Maybe be able to set up a delay (i.e. Show photo, 3 seconds later show first comment, then 3 seconds later, next comment, etc.). Not sure how that would work with the refresh time, but maybe it's a quick calculation to say "15 second refresh time, and there are 5 comments, so show one comment every 3 seconds" or something?
Thanks for listening to my rant!

**Comment by damongolding** [comment]
I’m not going to lie. I didn’t even know you could comment in Immich!

I will have an investigate about how easy it is to access those comments.

**Reply by m-a-johnson** [reply]
Yea, it isn't a super obvious feature and I think it only appears when you have a shared album.

**Reply by m-a-johnson** [reply]
just an FYI, comments can be pulled from the API. I just tested using curl as below (redacted). I am definitley NOT a programmer and wouldn't know how to implement, but thought it may be useful. If it isn't, feel free to ignore!

`curl -L "[redacted]/api/activities?albumId=[redacted]" -H "Accept: application/json" -H "x-api-key: [redacted]" | jq`

Output looks like below:

```
[
  {
    "id": "[redacted]",
    "assetId": "[redacted],
    "createdAt": "2025-01-08T18:42:41.524Z",
    "comment": "test comment",
    "type": "comment",
    "user": {
      "id": "[redacted]",
      "email": "[redacted]",
      "name": "[redacted]",
      "profileImagePath": "upload/profile/[redacted]/[redacted]png",
      "avatarColor": "purple",
      "profileChangedAt": "2024-12-06T21:04:33.915Z"
    }
  }
]
```
